ios/Podfile.lock

```diff
@@ -1023,4 +1023,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 2b38dc4e50f6b19e7a7db1e41739f3209e9d05e6
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.10.0
```

Comment:
seen this bouncing back and forth between versions

Reply:
I dont know which one is right

Reply:
I'll make it fixed to .10 later